asab/metrics/storage.py

```diff
@@ -0,0 +1,37 @@
+from .openmetric import metric_to_text
+from .watch import watch_table
+
+
+class MetricsDataStorage(object):
+
+	def __init__(self):
+		self.Tree = dict()
+
+
+	def create_metric_storage(self, dimension: str):
+		if dimension in self.Tree:
+			raise RuntimeError("Metrics dimension already exists in the data storage")
+
+		data = dict()
+		self.Tree[dimension] = data
+		return data
+
+
+	def get_all(self):
+		return self.Tree.values()
+
+
+	def get_all_in_openmetric(self):
```

Comment:
This method belongs into handler `asab.api.handler` - or in a dedicated handler in the `asab.metrics` that is invoked from `asab.api` for a sake of better colocation in the code.